label_studio/projects/migrations/0028_auto_20241107_1031.py

```diff
@@ -0,0 +1,86 @@
+# Generated by Django 4.2.16 on 2024-11-07 10:31
+
+from django.db import migrations
+import logging
+
+from core.models import AsyncMigrationStatus
+from core.redis import start_job_async_or_sync
+
+logger = logging.getLogger(__name__)
+migration_name = '0028_auto_20241107_1031'
+
+
+def forward_migration(migration_name):
+    from django.db.models import Count, Min
```

Comment:
seems a bit strange to import these here? not sure also if the import below needs to be done when the function runs